ASUWebPlatforms/webspark-ci
# WS2-1528: Add additional fields for configuring media items of type 'image'

Ticket: https://asudev.jira.com/browse/WS2-1528

QA Steps:

- Add a 'Text Content' component.
- Insert an image using the 'Insert from Media Library' option within the WYSIWYG editor.
- Insert an image of type 'Image'.
- Click on the "Edit media" button that appears on the image within the WYSIWYG editor.
- Configure the new fields that appear in the dialog box (Spacing top, Spacing right, Spacing bottom, Spacing left, Round image).
- Verify that modifying these fields updates the styles of the inserted image.
